src/page/gjennomforing-detaljer/endringsmeldinger/endringsmelding/useVarighet.ts

```diff
@@ -0,0 +1,22 @@
+import { useEffect, useState } from 'react'
+
+export const useVarighet = (key: string, defaultVerdi: number): [number, (v:number) => void] => {
```

Comment:
Ville kanskje endret navnet litt til `useLagretVarighet` eller så kunne man laget en generisk hook for å lagre med localStorage. Kan også vurdere om man trenger å sende inn key hvis hooken kun skal brukes for varighet

Reply:
Enig, vi vurderte litt rundt det å lage en generisk `useLocalStorage` hook, og det kan kanskje gjøres senere.